pkg/apis/pipeline/v1beta1/taskrun_conversion_test.go

```diff
@@ -78,13 +78,15 @@ func TestTaskrunConversion(t *testing.T) {
 					Name: "param-task-1",
 					Value: v1beta1.ParamValue{
 						ArrayVal: []string{"value-task-1"},
+						Type:     "string",
```

Comment:
Do you know why tests passed before you added this?

Reply:
I think previously these remain empty string for `paramSpec` and `paramValue` so the round trip would pass.

Reply:
And this also made wonder if we need extra tests with the empty string for paramType?

Reply:
yes, I think we do need a new test case. I thought about it some more and I think your explanation is not quite right-- if the test input is an empty string, and the type cast is what's failing, ConvertTo would have failed. I think my original hypothesis in the bug was wrong; it's not the type cast that failed, it's that the output was type = "" which knative isn't able to marshal (serialize). Anyway your approach here is still correct, please just add the new test case.